src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom.py

```diff
@@ -38,9 +38,15 @@
 password_length = 15
 
 
-def create_vault(client, vault_name, region, resource_group_name):
+def create_vault(cmd, client, vault_name, resource_group_name, region=None):
     vault_sku = Sku(SkuName.standard)
     vault_properties = VaultProperties()
+
+    # If region is not passed, create the vault in the same region as that of the resource group.
+    if not region:
+        vault_rg = resource_groups_cf(cmd.cli_ctx).get(resource_group_name)
+        region = vault_rg.location
```

Comment:
I would recommend renaming `region` to `location` and then just using the built in validator:
```Python
c.argument('location', validator=get_default_location_from_resource_group)
```